DMTR-53.tex

```diff
@@ -147,47 +147,297 @@ \subsection{Summary Table}
+
+One \texttt{pytest} error was encountered by \texttt{ap\_verify}:
+
+\begin{verbatim}
+______________________ MeasureRuntimeTestSuite.test_valid ______________________
+
+self = <test_profiling.MeasureRuntimeTestSuite testMethod=test_valid>
+
+    def test_valid(self):
+        """Verify that timing information can be recovered.
+            """
+        meas = measure_runtime(self.task.getFullMetadata(), task_name='isr', metric_name='ip_isr.IsrTime')
+        self.assertIsInstance(meas, Measurement)
+        self.assertEqual(meas.metric_name, lsst.verify.Name(metric='ip_isr.IsrTime'))
+>       self.assertGreater(meas.quantity, 0.0 * u.second)
+E       AssertionError: <Quantity 0.0 s> not greater than <Quantity 0.0 s>
+
+tests/test_profiling.py:54: AssertionError
+============================= 29 tests deselected ==============================\end{verbatim}
+
+This is a unit test limitation already reported 
+in \texttt{DM-12848} and does not affect the functionality of the package.
+
 \subsection{AG-00-05}
+
+\textit{Execution of the Alert Generation Pipelines produces the expected data products: PARTIALLY PASSED}
+
+We performed a Butler \texttt{get} for \texttt{calexp}, \texttt{deepDiff\_differenceExp}, and \texttt{deepDiff\_diaSrc} dataset types using 
+dataIds constructed all visits and ccds listed in the appendix of \citeds{LDM-533}.
+
+We found several dataIds where products were not generated:  
+
+\begin{itemize}
+	\item For visits 411321, 411371, and 411422, ccds 48--60 and 62 were missing all three of \texttt{calexp}, \texttt{deepDiff\_differenceExp}, and \texttt{deepDiff\_diaSrc} products.  This failure was due to the time limit being reached on the slurm execution script (which looped sequentially over ccds for each visit).
+	\item For visit 411657, ccd 47 only was missing all three products.  Inspecting the log revealed an error:
+		\begin{verbatim}  
+File "/software/lsstsw/stack3_20171023/stack/miniconda3-4.3.21-10a4fa6/
+Linux64/meas_algorithms/14.0-7-g23fdbe95+15/python/lsst/meas/algorithms/
+measureApCorr.py", line 251, in run
+    (name, len(subset2), self.config.minDegreesOfFreedom+1))
+RuntimeError: Unable to measure aperture correction for required algorithm 
+'base_PsfFlux': only 0 sources, but require at least 2.
+\end{verbatim}
+\end{itemize}
+
+We inspected the \texttt{DIAObject} table by opening the 
+\texttt{l1db/association.db} sqlite database produced by \texttt{ap\_pipe} 
+with \texttt{sqlite3}.  Issuing the query \texttt{select * from dia\_objects limit 10;} confirmed that the database was non-empty.
+
+
 \subsection{AG-00-10}
+
+\textit{Generated PVIs include science, mask, and variance images; background model; zeropoint; PSF, and WCS: PASSED}
+
+For all of the generated PVIs, we used Butler calls to check for the existence
+image, mask, and variance planes; zeropoints, PSFs, and WCS; and background models.
```

Comment:
In the LDM-503-2 test report, we quibbled that the background model is not actually stored as part of the PVI but is persisted separately. I don't think this is fundamentally important, but perhaps we should be consistent and raise the same concern here.